markdown/org/docs/patterns/otis/needs/en.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Otis, the simple onesie: What You Need"
+---
+
+To make Otis, you will need the following:
+
+- [Basic sewing supplies](/docs/sewing/basic-sewing-supplies)
+- About 1 meter (1.1 yards) of a suitable fabric ([see Otis Fabric options](/docs/patterns/otis/fabric/))
+- 3 snaps.
```

Comment:
The instructions say 3 snaps, but there are only 2 on the pattern?

Reply:
If I put the middle one on, it extends beyond the fold line. I decided that it looked too wrong to leave in place.